service/src/main/java/uk/gov/hmcts/reform/fpl/enums/ccd/fixedlists/InterimEndDateType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.fpl.enums.ccd.fixedlists;
+
+public enum InterimEndDateType {
+    END_OF_PROCEEDINGS,
+    NAMED_DATE
```

Comment:
I think we might need a value for the constants. This is so we can do `END_OF_PROCEEDINGS.getLabel()` in place of just returning the string as done in `GeneratedOrderService.getInterimExpiryDate()`.